File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03294/p03294_num0_parsed.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    // Read input
    int N;
    cin >> N;
    vector<int> a(N);
    for (int i = 0; i < N; i++) {
        cin >> a[i];
    }

    // Find maximum value of f
    int max_f = 0;
    for (int m = 0; m < a[0]; m++) {
        int sum = 0;
        for (int i = 0; i < N; i++) {
            sum += m % a[i];
        }
        max_f = max(max_f, sum);
    }

    // Print the maximum value of f
    cout << max_f << endl;

    return 0;
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03294/p03294_num0_parsed.py
import sys

def max_f(N, a):
    max_val = 0
    for m in range(max(a), max(a)*2):
        f = sum(m % ai for ai in a)
        max_val = max(max_val, f)
    return max_val

if __name__ == "__main__":
    N = int(input())
    a = list(map(int, input().split()))

    print(max_f(N, a))
    sys.stdout.flush()